### DISCUSSION TITLE
Artisan Bug

### DISCUSSION THREAD

**Discussion by Kitadrian** [artisan-roaster-scope/artisan#1675] [Q&A] [answered]
I'm using Kaleido Sniper M10 with updated artisans, and every time I'm putting a background, the SV can't be moved and stuck, please help , I've tried restarting the app, the Pc and even reinstalling the app

https://github.com/user-attachments/assets/f9a88429-b41f-40a5-b875-41699582769c

**Comment by MAKOMO** [comment, marked as answer]
This is by design. I guess your PID mode is set to "Background" (follow mode). In this mode the SV is taken from the background curve. As you did not CHARGE yet, the PID takes the BT temperature of the background curve as SV target.

You can set the PID mode to "Manual" to get full control over the SV via the SV slider (press the CONTROL button to open the PID config dialog).

<img width="685" alt="Screenshot 2024-08-19 at 15 19 08" src="https://github.com/user-attachments/assets/4c2ac559-eb52-4d27-9f7b-331034da6c8c">

You can also define custom buttons that allow you quickly toggle between background and manual mode like this.

<img width="933" alt="Screenshot 2024-08-19 at 15 19 56" src="https://github.com/user-attachments/assets/c26f97e1-b5a9-4c03-a751-b6bc4a86c75b">

**Reply by Kitadrian** [reply]
It worked! Thank you so much! Big help!
